tests/testthat/test-utils.R

```diff
@@ -184,3 +184,15 @@ test_that("assert_sufficient_f_args alerts if the provided f has defaults for th
                regexp = "window data and group key to `f`'s x and setting argument",
                class = "epiprocess__assert_sufficient_f_args__required_args_contain_defaults")
 })
+
+test_that("computation formula-derived functions take all argument types", {
+  # positional
+  expect_identical(as_slide_computation(~ ..2 + ..3)(1, 2, 3), 5)
+  expect_identical(as_slide_computation(~ ..1)(1, 2, 3), 1)
+  # Matching rlang, purr, dplyr usage
+  expect_identical(as_slide_computation(~ .x + .z)(1, 2, 3), 4)
+  expect_identical(as_slide_computation(~ .x + .y)(1, 2, 3), 3)
+  # named
+  expect_identical(as_slide_computation(~ . + .ref_time_value)(1, 2, 3), 4)
+  expect_identical(as_slide_computation(~ .group_key)(1, 2, 3), 2)
+})
```

Comment:
suggestion: round out the tests here with some cases where `as_slide_computation` produces errors.